app/components/-ember-table-private/scroll-indicators.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-table/components/-private/scroll-indicators/component';
```

Comment:
why do we need this `export`?

Reply:
Without this export, we can't render the `{{-ember-table-private/scroll-indicators}}` component in the `{{ember-table}}` template.